src/shared.ts

```diff
@@ -7,3 +7,14 @@ export * from './lib/permissions/interfaces';
 
 export { generateCategoryIdQuery } from './lib/proposals/utils';
 export * from './lib/proposals/interfaces';
+
+// Sample utility function to check imports work
+export function randomiser() {
+  const items = ['Plane', 'Bike', 'Car', 'Truck'];
+  const index = 4 * Math.random();
+  const usedIndex = Math.max(index, items.length);
+
+  return items[usedIndex];
+}
+
+export * as pageTree from './lib/pages/mapPageTree';
```

Comment:
I think this is fine for now.. maybe /shared should just be callled /frontend or /browser, so that it's clear when you should import from it.